frontend/src/pages/modelRegistry/screens/utils.ts

```diff
@@ -132,12 +136,12 @@ export const filterModelVersions = (
       case SearchType.KEYWORD:
         return (
           mv.name.toLowerCase().includes(searchLower) ||
-          (mv.description && mv.description.toLowerCase().includes(searchLower)) ||
+          mv.description?.toLowerCase().includes(searchLower) ||
```

Comment:
Same deal here, I'm not sure why we're removing the `?.` operator in unrelated code here, let's revert the changes from this line to the end of the file.